.circleci/run.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+
+case="master.A_WCYCL1850.ne4_oQU240.baseline"
+compset="A_WCYCL1850"
+res="ne4_oQU240"
+
+log_dir="${HOME}/projects/e3sm/scratch/$case"
+
+shopt -s extglob
+shopt -s nullglob
+
+function command_yellow
+{
+    echo -e "\e[33m\$ $1\e[0m"
+    eval $1
+    return $?
+}
+
+function command_yellow_rc
+{
+    command_yellow "$1"
+    rc=$?
+    if [ $rc -ne 0 ]; then
+        exit $rc
+    fi
+}
+
+function print_bldlog
+{
+    bldlog=$1
+    bldlog_path=(${log_dir}/bld/$bldlog.bldlog.+([[:digit:]])-+([[:digit:]]))
+    if [ -n "${bldlog_path}" ]; then
+        command_yellow "cat ${bldlog_path}"
+    fi
+}
+
+function print_runlog
+{
+    runlog=$1
+    runlog_path=(${log_dir}/run/$runlog.log.+([[:digit:]])-+([[:digit:]]))
+    if [ -n "${runlog_path}" ]; then
+        command_yellow "cat ${runlog_path}"
+        return 1
+    else
+        return 0
+    fi
+}
+
+command_yellow_rc "mkdir -p $HOME/projects/e3sm/cesm-inputdata"
+command_yellow_rc "cd cime/scripts"
+command_yellow_rc "./create_newcase --case $case --compset $compset --res $res"
+command_yellow_rc "cd $case"
+command_yellow_rc "./case.setup"
+command_yellow "./case.build"
```

Comment:
Since this was activated, many PRs are failing the build check here due to the 1-hour timeout. Looking closer, the default build parallelism (`GMAKE_J` in `env_run.xml` is 16 for machine `singularity`). After I ssh'ed into circle-ci instance, available number of cores is just two:
```
circleci@default-407fb855-d9b2-45be-8650-d5e693752cbf:~/project/cime/scripts/master.A_WCYCL1850.ne4_oQU240.baseline$ cat /proc/cpuinfo
processor       : 0
vendor_id       : GenuineIntel
cpu family      : 6
model           : 63
model name      : Intel(R) Xeon(R) CPU @ 2.30GHz
stepping        : 0
microcode       : 0x1
cpu MHz         : 2300.000
cache size      : 46080 KB
physical id     : 0
siblings        : 2
core id         : 0
cpu cores       : 1
apicid          : 0
initial apicid  : 0
fpu             : yes
fpu_exception   : yes
cpuid level     : 13
wp              : yes
bugs            : cpu_meltdown spectre_v1 spectre_v2 spec_store_bypass l1tf mds swapgs
bogomips        : 4600.00
clflush size    : 64
cache_alignment : 64
address sizes   : 46 bits physical, 48 bits virtual
power management:

processor       : 1
vendor_id       : GenuineIntel
cpu family      : 6
model           : 63
model name      : Intel(R) Xeon(R) CPU @ 2.30GHz
stepping        : 0
microcode       : 0x1
cpu MHz         : 2300.000
cache size      : 46080 KB
physical id     : 0
siblings        : 2
core id         : 0
cpu cores       : 1
apicid          : 1
initial apicid  : 1
fpu             : yes
fpu_exception   : yes
cpuid level     : 13
wp              : yes
bugs            : cpu_meltdown spectre_v1 spectre_v2 spec_store_bypass l1tf mds swapgs
bogomips        : 4600.00
clflush size    : 64
cache_alignment : 64
address sizes   : 46 bits physical, 48 bits virtual
power management:
```

@lukaszlacinski, please change that to 1 or 2, just before or after `./case.setup`:
```
./xmlchange GMAKE_J=2
```
Otherwise, build will continue hanging.

Or update this in config_machines.xml:
```
<GMAKE_J>2</GMAKE_J>
```